pkg/cmd/server/api/types.go

```diff
@@ -1407,3 +1408,16 @@ type DefaultAdmissionConfig struct {
 	// Disable turns off an admission plugin that is enabled by default.
 	Disable bool
 }
+
+// TemplateServiceBrokerConfig holds information related to the template
+// service broker
+type TemplateServiceBrokerConfig struct {
```

Comment:
@smarterclayton having this in process with openshift master is likely to hinder efforts to run on top of raw kube.  Are you making sure that we don't have any client that assumes, "oh the kube-apiserver has this endpoint for me" so that we can later transition it out?